spark/src/main/kotlin/com/adevinta/spark/components/chips/Chip.kt

```diff
@@ -326,6 +335,9 @@ public fun ChipSelectable(
                         color = contentColor,
                     )
                 }
+//                // Fix to prevent chip from having a width of zero when used
+//                // in a scroll container
```

Comment:
same

Reply:
Done in [fe8c41c](https://github.com/adevinta/spark-android/pull/1236/commits/fe8c41c37836b6173f6030041f5959f7bfb6ab08)